packages/connect-ui/src/lib/i18n/translations/de.ts

```diff
@@ -0,0 +1,39 @@
+const de = {
```

Comment:
I think you meant this no?
```suggestion
const de: Translation = {
```

Reply:
Indeed